### ISSUE TITLE
Undesired interpretation of HTML-encoded characters

### ISSUE THREAD

**Issue by Xavier-Mobius** [SebastianStehle/mjml-net#192]
Hello, and thank you for this very useful library! :)

Mjml.Net does not produce the same result as mjml.io. 
Mjml.Net interprets HTML-encoded characters (it "de-encodes" such characters) even if they are "HTML Character Entity Number" instead of "Character Entity Names"..

A very simple example:
```C#
using Mjml.Net;

var mjmlRenderer = new MjmlRenderer();
(var result, var errors) = mjmlRenderer.Render(@"<mj-text>Hello &gt;&#62; World</mj-text>");

Console.WriteLine(result);
```

Output:
```html
<div style="[...]">
Hello >> World
</div>
```

The output of the same MJML in https://mjml.io/try-it-live/eiXRjriBbY :
```html
<div style="[...]">Hello &gt;&#62; World</div>
```

How can I fix this, or get around this problem?

**Comment by SebastianStehle** [comment]
I had a look to it and is not resolved properly at the moment. I created a PR for the html library: https://github.com/osjoberg/HtmlPerformanceKit/pull/8

I hope this gets merged (soon). Then I can provide a fix for you.

**Comment by SebastianStehle** [comment]
Sorry, I have forgotten to notify. This was actually merged in a week ago or so.

**Comment by Xavier-Mobius** [comment]
Yes, I saw that yesterday, you fixed it in the version 3.9. Thanks a lot :)
